src/sentry/relocation/tasks/process.py

```diff
@@ -1194,6 +1255,10 @@ def camel_to_snake_keep_underscores(value):
     retry_backoff_jitter=True,
     soft_time_limit=FAST_TIME_LIMIT,
     silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=relocation_tasks,
```

Comment:
Should we have a processing_deadline of FAST_TIME_LIMIT to align with the celery configuration?